00-HTML-CSS-basics/Practica1-2.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang = "en">
+  <head>
+    <meta charset="utf-8">
+    <title> Table of expenses </title>
+  </head>
+  <body>
+    <table>
+      <tr>
+        <th>Yours expenses</th>
+        <td>Expenses</td>
+        <td>Month</td>
+        <td>State</td>
+        <td>Mount</td>
+      </tr>
+      <tr>
+        <td>Epe</td>
+        <td>September</td>
+        <td>paid out</td>
+        <td>1200</td>
+      </tr>
+      <tr>
+        <td>LitoralGas</td>
+        <td>September</td>
+        <td>paid out</td>
+        <td>700</td>
+      </tr>
+      <tr>
+        <td>Aguas</td>
```

Comment:
Whit "aguas" i refer to "aguas provinciales" ( the company), its not clear, sorry.